nengo/spa/input.py

```diff
@@ -40,7 +40,10 @@ def on_add(self, spa):
         Module.on_add(self, spa)
 
         for name, value in iteritems(self.kwargs):
-            target, vocab = spa.get_module_input(name)
+            try:
+                target, vocab = spa.get_module_input(name)
+            except TypeError:
+                raise ValueError('Could not find input called "%s"' % name)
```

Comment:
Yup, I'd be onboard with changing the ```spa.get_*``` stuff so that they raise exceptions rather than returning None (which is what they do now).  Should I add that to this PR or start a new one?